### PR TITLE
A few additional notes to assist with adding new spectrographs

### PR DESCRIPTION
**Pull request by profxj**
As titled.

Mainly notes on tests and docs.

### CODE REVIEW

**Review by jhennawi — APPROVED** [review]

**Comment by profxj** [comment]
(base) profx> py.test test_load_images.py 
============================= test session starts ==============================
platform linux -- Python 3.8.5, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /data/Projects/Python/PypeIt, configfile: setup.cfg
plugins: ligo.skymap-0.5.0
collected 12 items                                                             

test_load_images.py ............                                         [100%]

=============================== warnings summary ===============================
../../../../../../home/xavier/Projects/anaconda3/lib/python3.8/site-packages/pkg_resources/__init__.py:1143
../../../../../../home/xavier/Projects/anaconda3/lib/python3.8/site-packages/pkg_resources/__init__.py:1143
../../../../../../home/xavier/Projects/anaconda3/lib/python3.8/site-packages/pkg_resources/__init__.py:1143
   DeprecationWarning: Use of .. or absolute path in a resource path is not allowed and will raise exceptions in a future release. (__init__.py:1143)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================== 12 passed, 3 warnings in 6.62s ========================

File: doc/new_spectrograph.rst [hunk 1]
```diff
@@ -95,4 +95,23 @@ If this is a near-IR instrument, you may wish to turn off calibration steps.
...
+- A unit test in ``test_load_images.py`` to tickle the I/O.
+
+Docs
+++++
+
+We request that the following docs be updated to advertise the new
+spectrograph:
+
+- The top-level ``README`` file
+- The list of spectrographs in ``doc/include/spectrographs_table.rst``
```
Comment (kbwestfall) [inline doc/new_spectrograph.rst]: This is automatically generated by `doc/scripts/build_spectbl_rst.py` so this shouldn't be edited.  Just make sure the spectrograph module is imported in `pypeit/spectrographs/__init__.py`.

**Review by kbwestfall — APPROVED** [review]
Looks good.  One minor edit to new_spectrograph.rst.

File: doc/new_spectrograph.rst (hunk 1 above)
Comment (profxj) [reply doc/new_spectrograph.rst]: removed

**Pull request merged**